test/bdd/steps/api/search-assertion.js

```diff
@@ -0,0 +1,68 @@
+const { When, Then, Given } = require('@cucumber/cucumber');
+const { expect, assert } = require('chai');
+const utilities = require('../../../utilities/utilities');
+let handlerId, searchResult, searchData, parsedKeywords, lastSearchData;
+
+When(/^I call search request on node (\d+) with result type ([^"]*) for the keywords:*$/, { timeout: 120000 }, async function (node, resultType, keywords) {
+    parsedKeywords = utilities.unpackRawTableToArray(keywords);
+    const result = await this.state.nodes[node - 1].client
+        .search(resultType, parsedKeywords[0]).catch((error) => {
+            assert.fail(`Error while trying to search assertion. ${error}`);
+        });
+    expect(result.status).to.be.eql(202, 'Expected search result status is 202');
+    expect(result.statusText).to.be.eql('Accepted');
```

Comment:
For every expect please add a comment if check fails so it's easier to track down the logs, i.e.:
expect(result.statusText).to.be.eql('Accepted', 'Expected status text of search assertion result to be accepted');